desc/compute/_core.py

```diff
@@ -2118,6 +2118,7 @@ def _lambda_zzz(params, transforms, profiles, data, **kwargs):
 )
 def _omega(params, transforms, profiles, data, **kwargs):
     data["omega"] = data["0"]
+    # TODO: what does omega !=0 mean for ZernikeRZToroidalXS?
```

Comment:
It means a ZernikeRZToroidalXS cross-section for a (rho, theta, zeta) grid is no longer a constant phi cut in the lab frame. A ZernikeRZToroidalXS cross-section for a (rho, theta, phi) grid remains a constant phi cut in the lab frame.